ui/app/AppLayouts/Wallet/views/collectibles/CollectibleView.qml

```diff
@@ -18,6 +18,7 @@ Control {
     property string title: ""
     property string subTitle: ""
     property alias subTitleColor: subTitleItem.customColor
+    property alias tokenCommunityTagVisibility : manageTokenCommunityTag.visible
```

Comment:
I'm not sure about adding this extra property here since the visibility of the `ManageTokensCommunityTag` is already binded with `isCommunityCollectible` property.

Would it make sense to just don't set `communityId` on `MintedTokensView` line **257**?